pegasus/sites.v3/code.org/public/css/learn-carousel.css

```diff
@@ -200,16 +175,36 @@
     width: 185px;
   }
 
-  .multi-item-carousel .carousel-inner .active.left {
-    left: -0%;
+  #banner-slideshow {
+    height: 80px;
   }
-  .multi-item-carousel .carousel-inner .active.right {
-    left: 0%;
+
+  @media all and (transform-3d), (-webkit-transform-3d) {
+    .carousel-inner > .item.next,
+    .carousel-inner > .item.active.right {
+      left: 0;
+      -webkit-transform: translate3d(50%, 0, 0);
```

Comment:
is it okay that translate3d not available in ie8/9 and only partially in ie10/11?